lib/record-serialization-fallback.js

```diff
@@ -261,8 +258,10 @@ JsrsParser.prototype.skipComments = function() {
 // Skip the body of a single-line comment (i.e. the part after //)
 //
 JsrsParser.prototype.skipLineCommentBody = function() {
-  while (!this.isNewline(this.lookahead()) &&
-          this.lookahead() !== undefined) {
+  while (
```

Comment:
If we stick braces with the code and not put them on separate lines then this cycle should not be changed.